index.js

```diff
@@ -32,7 +33,7 @@ CacheBreaker.prototype.cacheBreakPath = function(base, resource) {
   var extname = path.extname(resource);
   var basename = path.basename(resource, extname);
 
-  return path.join(dirname, basename + '.' + cs.substring(0, 10) + extname);
+  return url.resolve(dirname + '/', basename + '.' + cs.substring(0, 10) + extname);
```

Comment:
Ah, good point.

In this case it's only intended to handle file system paths, not urls.   It sounds like you're trying to cache-break a remote resource referenced by url?  If so we'll need to patch the library to make that possible (but it seems like a totally do-able and ideal feature).